packages/cozy-sharing/src/DoctypeContact.js

```diff
@@ -1,27 +1,28 @@
+import { models } from 'cozy-client'
```

Comment:
For my information, what was the warning that was solved by using cozy-client models ?

Reply:
Commit edited but "Replace deprecated cozy-doctypes/contactModel methods"